### PR TITLE
Create PRIVACY_POLICY.md

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR creates a Privacy Policy for the whole org, or as referred in the document the Pulsar Edit Project.

In it, I've detailed the privacy policy as it relates to:

* The Pulsar Application
* The Pulsar Main Website
* The Pulsar Backend
* A Pulsar User Account (Which isn't yet available, but will be soon on the backend)

Please feel free to give this a thorough review, especially if any information is found to be inaccurate.

I've also added a last changed date in the document, that **MUST** be updated any time the document is edited.

This document Ideally could then be included on our main website to allow easy viewing, and consumption. And allowing a much prettier link.

### CODE REVIEW

File: PRIVACY_POLICY.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+### What Information is Collected
+
+* GitHub Username
+* GitHub Gravatar URL
+* GitHub `node_id` (Think of your `node_id` like the random number assigned to you when making a user account. This is public information)
+
+### How is this Information Collected
+
+When you sign up with Pulsar, and choose to make an account, this is the data collected to make sure your account works, and is able to protect Packages from malicious
+Actors pretending to be someone they're not.
```
Comment (kyjus25) [inline PRIVACY_POLICY.md:65]: Is this supposed to be a new line?

File: PRIVACY_POLICY.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### What is Done with this Information
+
+This information is used to let you authenticate against the Pulsar Backend, when publishing, deleted, or starring a package.
```
Comment (kyjus25) [inline PRIVACY_POLICY.md:69]: "...against the Pulsar backend when publishing, deleting, or starring a package."

File: PRIVACY_POLICY.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+The only people who have access to this information are Pulsar's Core Admin Team in charge of the Backend.
+
+### How Long is this Information Kept
+
+The above details will be kept until either you request deletion of your account, or there is a built in supported method to delete your Pulsar User Account.
+
+### How do I have this Information Deleted
+
+To request removal of the above details feel free to contact the Pulsar Core Admin Team, through any of the [supported methods](https://pulsar-edit.dev/community.html).
+But the recommended method would be to contact @Admins on the [Pulsar Discord](https://discord.gg/7aEbB9dGRT) or via email to admin@pulsar-edit.dev
```
Comment (kyjus25) [inline PRIVACY_POLICY.md:82]: Remove "but", perhaps put it above the first one if it is the recommended method. Maybe say instead "for quicker response times, contact through Discord"

File: PRIVACY_POLICY.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+* The only people who can access this information is the Pulsar Core Admin Team.
+* This information is purely there to help you use the Pulsar Backend to publish, delete, star, or otherwise interact with Packaages.
+* Pulsar User Accounts contain **Zero** details of how you log in, or any type of API, OAuth, or other token to access your account.
+
+## Pulsar Website (`https://pulsar-edit.dev`)
+
+The Pulsar Website is a service you connect to anytime you visit our website.
+
+This Website is hosted on [GitHub Pages](https://pages.github.com/) which while conveinent for us, also means that we have no built in mechanism to access any personal data of any kind through this service.
+That is to say we don't, no analytics are setup on the website, and no extra code is run on the website to collect any data.
```
Comment (kyjus25) [inline PRIVACY_POLICY.md:97]: "Setup" -> "set up"

File: PRIVACY_POLICY.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+* The Pulsar Website Collects **Zero** Data about you, or where you are.
+* It does not Log anything, save anything, or extract anything.
+* It's just a basic website.
+
+## Pulsar Editor
+
+The main Pulsar Application, available [here](https://github.com/pulsar-edit/pulsar) is the program you use anytime you launch Pulsar, or edit text within it.
+
+While [Atom](https://atom.io) the original creator of the Editor did collect telemetry and metrics about the users, when we began work on the editor one of the first
```
Comment (kyjus25) [inline PRIVACY_POLICY.md:109]: "While GitHub, the original creator of the editor, did collect telemetry..."

**Review by kyjus25 — COMMENTED** [review]
My two cents if you're interested

File: PRIVACY_POLICY.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
+# Pulsar Edit Privacy Policy
+
+The team on Pulsar greatly values yours, and everyones privacy. In that respect we keep any collection of personal data to an absolute minium,
+but for what little is collected across the entirety of the Pulsar Edit Project will be detailed here.
```
Comment (Daeraxa) [inline PRIVACY_POLICY.md:4]:
```suggestion
but what little is collected across the entirety of the Pulsar project will be detailed here.
```

**Review by Daeraxa — COMMENTED** [review]
Just added some suggestions, feel free to disagree.

File: PRIVACY_POLICY.md (hunk 5 above)
Comment (Daeraxa) [reply PRIVACY_POLICY.md:109]: This one I think is ambiguous as the repo organisation is indeed Atom even if it is GH behind that. Maybe:
> While Atom, the original version(/implementation?) of the editor, did collect...
?

**Comment by Daeraxa** [comment]
Also, do you know if we have definitely removed all code that goes to third parties? For example is there anything that is capable of reaching out to npm?
I know that the `exception-reporting` package is designed to report data to https://bugsnag.com, has that mechanism been entirely disabled?

File: PRIVACY_POLICY.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (confused-Techie) [reply PRIVACY_POLICY.md:97]: Love how proper your writing is, great catch that I wouldn't have seen on a million read throughs lol

**Comment by Sertonix** [comment]
Things like `teletype` and other packages that make web requests are not listed here. It would make sence to note that third party packages are not under our controll and can do anything. For the packages by Pulsar-Edit I would suggest including it here or add a link to a `PRIVACY_POLICY.md` in there repo.

**Comment by Sertonix** [comment]
Also the [town-crier](https://github.com/pulsar-edit/town-crier) package but it is the same as with teletype. Alsmost unmaintaind.

**Comment by confused-Techie** [comment]
Alright added a notice that we can't vouch for community packages, @Daeraxa look good to you?

**Comment by confused-Techie** [comment]
Ignore my misclick there lol

**Review by Sertonix — APPROVED** [review]

**Review by Daeraxa — COMMENTED** [review]
A couple more things I noticed of a read through

File: PRIVACY_POLICY.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+Also please note that this Privacy Policy excludes any Community Packages that may be installed as those are created and maintained by a third party.
+
+At the bottom of each service listed in this document there will be a "TLDR" to quickly sum up all data that is collected, how it's used, and who can use it.
+
+Date of the Last Change to the Privacy Policy:
+2022, December 10th
+
+## Pulsar Backend (`https://api.pulsar-edit.dev`)
+
+The Pulsar Backend is the service you connect to when using all Pulsar Packages on the web via `https://web.pulsar-edit.dev` and when using the built in Pulsar [Settings View](https://github.com/pulsar-edit/settings-view) or [Package Manager](https://github.com/pulsar-edit/ppm) to interact with community packages.
```
[the remark this answers is not in the document]
Comment (confused-Techie) [reply PRIVACY_POLICY.md:15]: No I think you're right, it might be best to remove this one

**Comment by confused-Techie** [comment]
Thanks for checking again @Daeraxa, lemme know if I've missed anything

**Comment by Daeraxa** [comment]
Looks good to me now! Not spotted anything else.

**Pull request merged**